index.js

```diff
@@ -43,6 +48,9 @@ module.exports = (input, opts) => {
 
 module.exports.sync = (input, opts) => {
 	opts = Object.assign({cwd: process.cwd()}, opts);
+	if (typeof opts.cwd !== 'string') {
+		throw new TypeError('opts.cwd must be a string');
```

Comment:
```js
new TypeError(`Expected `cwd` to be of type \`string\` but received type \`${typeof opts.cwd}\``);
```

Reply:
error message updated, all the other changes made the project pass the lint by xo, pin `ava` and `xo` to ensure it work in node v4.

Reply:
I've make these changes into two commit if you want to separate them.